### ISSUE TITLE
Can’t open app.

### ISSUE THREAD

**Issue by Gamerboi276** [zzanehip/The-OldOS-Project#81]

**Comment by zzanehip** [comment]
Are you on the iOS 16 beta? I know I encountered this issue on the beta, and believe it is a bug. The fix is to turn on developer mode (for now, until its resolved).

**Comment by Gamerboi276** [comment]
> 

Nope, I’m on iPadOS 15, and I can’t find the developer mode option, should I install the iOS 16 beta for now?

**Comment by LilKitteNebula** [comment]
You'll have to sign the app, usually 3uTools works good for me. I think it's available for both Windows and Mac

**Comment by Gamerboi276** [comment]
Dang, I use only an iPad running iOS 15.5* and altstore and other Sideloaders don’t work for me. Think altstore’s error was to update xcode packages to 7.3, I don’t really remember

**Comment by langonginc** [comment]
Is there any certificate free for me? @LilKitteNebula

**Comment by ripjaw1219** [comment]
bro i have this same issue help
